ledger/controllers/wallet.js

```diff
@@ -254,6 +254,13 @@ v2.read = { handler: (runtime) => { return read(runtime, 2) },
    PUT /v2/wallet/{paymentId}
  */
 
+function voteValueFromSurveyor (surveyor) {
+  const { votes, probi } = surveyor.payload.adFree
+  const bigProbi = new BigNumber(probi)
+  const minimum = bigProbi.dividedBy(votes).dividedBy(1e18).toString()
```

Comment:
to be consistent we shouldn't just hardcode 1e18 here and instead should use the runtime currency facilities

Reply:
i was thinking about this yesterday and forgot to get to it this morning. thanks!